.travis.yml

```diff
@@ -10,9 +10,14 @@ sudo: required
 services:
   - docker
 
-# Install molecule
+env:
+  - ANSIBLE='ansible>=2.3.0,<2.4.0'
+  - ANSIBLE='ansible>=2.4.0,<2.5.0'
+  - ANSIBLE='ansible>=2.5.0,<2.6.0'
+
+# Install ansible, molecule, and dependencies
 install:
-  - pip install -r test-requirements.txt
+  - pip install ${ANSIBLE} 'molecule>=2.15.0' docker 'ansible-lint>=3.4.0' 'testinfra>=1.7.0'
```

Comment:
I think the docker package already includes molecule now, we can probably drop that as well.

Reply:
done

Reply:
Docker package includes molecule? That doesn't seem to be right.

Reply:
Sorry, the images used for testing, ie. `image: paulfantom/centos-molecule` already include it.

Reply:
But `docker` is needed by ansible to provision this image

Reply:
No, I meant removing the molecule package, not the docker package. But I guess those are not included in the base travis image. 

Nevermind!

Reply:
Yeah, molecule isn't installed by default in travis.